sound/soc/sof/ops.h

```diff
@@ -244,15 +244,19 @@ snd_sof_dsp_set_power_state(struct snd_sof_dev *sdev,
 static inline void snd_sof_dsp_dbg_dump(struct snd_sof_dev *sdev, u32 flags)
 {
 	if (sof_ops(sdev)->dbg_dump && !sdev->dbg_dump_printed) {
+		dev_err(sdev->dev, "------------[ dsp dump start ]------------\n");
 		sof_ops(sdev)->dbg_dump(sdev, flags);
+		dev_err(sdev->dev, "------------[ dsp dump end ]------------\n");
```

Comment:
as you plan to update the series anyway - don't you think capitalising "DSP" and "IPC" below would be preferable?

Reply:
OK, I'll do that as well, thanks!